etl/steps/data/garden/health/2025-02-20/measles_deaths_long_run.py

```diff
@@ -0,0 +1,41 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from owid.catalog import processing as pr
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow_hist = paths.load_dataset("measles_deaths_historical")
+    ds_who_mort = paths.load_dataset("mortality_database_vaccine_preventable")
+
+    # Read table from meadow dataset.
+    tb_hist = ds_meadow_hist.read("measles_deaths_historical")
+    tb_pres = ds_who_mort.read("mortality_database_vaccine_preventable")
+    tb_pres = tb_pres[
+        (tb_pres["cause"] == "Measles")
+        & (tb_pres["age_group"] == "all ages")
+        & (tb_pres["country"] == "United States")
+        & (tb_pres["sex"] == "Both sexes")
+    ]
+    tb_pres = tb_pres[["country", "year", "number"]]
+    tb_pres = tb_pres.rename(columns={"number": "deaths"})
+
+    tb = pr.concat([tb_hist, tb_pres], short_name="measles_deaths_long_run", ignore_index=True)
```

Comment:
Hi @pabloarosado - do you have any insight into what I'm doing wrong here? Both tb_hist and tb_pres have metadata prior to this line, but it disappears after pr.concat().

Reply:
Hi @spoonerf the resulting concatenated table does have metadata. But I noticed it doesn't have a table title, which then causes an issue in the grapher step because "Dataset must have a title". This happens because you are putting together two origins with different titles, and hence the concatenated table doesn't know what to do, and simply removes the title. You can either manually add a dataset title in the meta.yml file, or add a `default_metadata` to `create_dataset`. Does that make sense?